actionview/test/template/dependency_tracker_test.rb

```diff
@@ -55,159 +55,176 @@ def test_returns_empty_array_if_no_tracker_is_found
   end
 end
 
-class RipperTrackerTest < Minitest::Test
-  def make_tracker(name, template)
-    ActionView::DependencyTracker::RipperTracker.new(name, template)
-  end
+# Tests run with both ERBTracker and RipperTracker
+module SharedTrackerTests
```

Comment:
We're only adding instance methods so there's no need to extend as a concern nor indent, let's use a standard module.